src/middlewared/middlewared/plugins/smb.py

```diff
@@ -549,6 +549,27 @@ def getparm(self, parm, section):
         except Exception as e:
             raise CallError(f'Attempt to query smb4.conf parameter [{parm}] failed with error: {e}')
 
+    @private
+    @job(lock="rid_check")
+    async def check_rid_conflict(self, job):
+        pdb_by_sid = {}
+        passdb_jobs = await self.middleware.call(
+            'core.get_jobs',
+            [('method', '=', 'smb.synchronize_passdb'), ('status', 'in', ['WAITING', 'RUNNING'])]
+        )
+        for pdb_job in passdb_jobs:
```

Comment:
get_jobs is not supposed to return Job objects so you cant wait on them like that

Reply:
@anodos325 ping

Reply:
You're correct. I'll fix it in a little bit :)